pyproject.toml

```
@@ -31,6 +31,10 @@ select = [
   "PLC", # pylint convention
   "PLE", # pylint error
   "Q",   # flake8-quotes
+  "A",   # flake8-builtins
+  "B",   # flake8-bugbear
+  "N",   # pep8-naming
+  "RUF", # Ruff-specific rules
```

I know this is following the pattern above, but can we add the links instead?

```suggestion
  "A",   # https://docs.astral.sh/ruff/rules/#flake8-builtins-a
  "B",   # https://docs.astral.sh/ruff/rules/#flake8-bugbear-b
  "N",   # https://docs.astral.sh/ruff/rules/#pep8-naming-n
  "RUF", # https://docs.astral.sh/ruff/rules/#ruff-specific-rules-ruf
```

It really makes the review/understanding on the rules while developing faster to just click the link.

That said... Are those rules just running on pre-commit? Or the changes on the files are not applied yet?

In any case... I think all those rules are pedantic. Maybe instead of randomly getting rules from the list, checking some known projects, and which rules they use?

---

This comment is not necessarily for this PR, it's just about the setup itself.